examples/server/esp32spi_wsgiserver.py

```diff
@@ -1,246 +0,0 @@
-# SPDX-FileCopyrightText: 2019 ladyada for Adafruit Industries
```

Comment:
Would it make sense to move these examples over to https://github.com/adafruit/Adafruit_CircuitPython_WSGI/tree/main/examples and modify if needed to work with that library? 

I'll test out this version of the library using the examples and a few of the basic projects from learn.

Reply:
We could do that, but I think that library could use some work as well. I don't see it as that high a priority. I don't think the old WSGI library here is used much.

I would be most interested in a subset of some standard or semi-standard CPython WSGI library, in the long run.